components/tyk-gateway/values.yaml

```diff
@@ -14,6 +14,11 @@ nameOverride: ""
 fullnameOverride: ""
 
 global:
+  tls:
+    gateway: false
```

Comment:
We also a field to enable tls under `tyk-gateway.tls.enabled`. This might create confusion